src/core/tests/type_prop/fake_convert.cpp

```diff
@@ -49,3 +70,55 @@ TEST(type_prop, fake_convert_dynamic_shape) {
     EXPECT_EQ(op->get_output_element_type(0), element::f32);
     EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape::dynamic()));
 }
+
+TEST(type_prop, fake_convert_basic_unsupported_type) {
+    const auto data = std::make_shared<Parameter>(element::f64, PartialShape{2, 3, 8, 6});
+    const auto scale = std::make_shared<Parameter>(element::f64, PartialShape{});
+    const auto shift = std::make_shared<Parameter>(element::f64, PartialShape{});
+
+    OV_EXPECT_THROW(
+        const auto op = std::make_shared<op::v13::FakeConvert>(data, scale, shift),
+        Exception,
+        testing::HasSubstr("The element type of the input tensor must be a bf16, f16, f32 or dynamic (got f64)."));
+}
+
+TEST(type_prop, fake_convert_basic_unsupported_shape_scale_shift) {
+    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 3, 8, 6});
+    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{1});
+    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{});
+
+    OV_EXPECT_THROW(const auto op = std::make_shared<op::v13::FakeConvert>(data, scale, shift),
+                    AssertFailure,
+                    testing::HasSubstr("FakeConvert scale shape: [1] is not compatible with shift shape: []"));
+}
+
+TEST(type_prop, fake_convert_basic_unsupported_shape_not_broadcastable) {
+    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 3, 8, 6});
+    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});
+    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});
+
+    OV_EXPECT_THROW(const auto op = std::make_shared<op::v13::FakeConvert>(data, scale, shift),
+                    AssertFailure,
+                    testing::HasSubstr("Argument shapes are inconsistent."));
+}
+
+TEST(type_prop, fake_convert_basic_unsupported_shape_not_unidirectional_broadcastable) {
+    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 3, 1, 6});
+    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});
+    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});
+
```

Comment:
Maybe just few more tests with correct examples and scale/shape without aligned rank or dynamic dimensions in range like:

```
    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 1, 3, 6});
    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});
    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{1, 1, 3, 1});


    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 1, 3, 6});
    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{3, 1});
    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{3, 1});


    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 1, {3, 5}, 6});
    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{3, 1});
    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{3, 1});

    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{4, {2, 5}, 6});
    const auto scale = std::make_shared<Parameter>(element::f32, PartialShape{4, 3, 1});
    const auto shift = std::make_shared<Parameter>(element::f32, PartialShape{4, 3, 1});
```
  And at least one test for labels of the data input (set_shape_labels helper set sequence of the labels starting from the provided number, it can be set to 10, 11, 12.. by set_shape_labels(data_pshape, 10), and checked if the outputs preserved the labels.
  Example:
   https://github.com/openvinotoolkit/openvino/blob/613e4f5fac13b726f26e51619f3d59e12a8302f6/src/core/tests/type_prop/scatter_elements_update.cpp#L38-L55

Reply:
Done